ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -0,0 +1,174 @@
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+#include <fstream>
+#include <list>
+#include <errno.h>
+#include <filesystem>
+#include <mutex>
+
+#include "gcp_connection.h"
+#include "gcp_log_system.h"
+
+#define UNUSED(x) (void)(x)
```

Comment:
We don't need the UNUSED macro, WT macro should already exist.

Reply:
yup, addressed here: [eca9030](https://github.com/wiredtiger/wiredtiger/pull/8688/commits/eca9030299a0be8f40407136ddd9dd7e4dee9560)